oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthCodeHandlerTest.java

```diff
@@ -314,7 +314,7 @@ void shouldGenerateSuccessfulAuthResponseAndUpliftAsNecessary(
                                 .withIpAddress("123.123.123.123")
                                 .withPersistentSessionId(PERSISTENT_SESSION_ID),
                         pair("internalSubjectId", SUBJECT.getValue()),
-                        pair("isNewAccount", AccountState.NEW),
+                        pair("isNewAccount", OrchSessionItem.AccountState.UNKNOWN),
```

Comment:
why has this changed from NEW to UNKNOWN?

Reply:
Purely based on setting the Orch session for the tests. I've updated the orch session which is setup to have the same account state (NEW) so this change looks a little less confusing 🙂